Application/Controller/Admin/Admin_oxpsModuleGenerator.php

```diff
@@ -366,7 +375,7 @@ protected function _getFormVersionFieldValue($sRequestVersion)
         $sModuleVersion = trim((string) $sRequestVersion);
 
         if (empty($sModuleVersion)) {
-            $sModuleVersion = '1.0.0';
+            $sModuleVersion = '2.0.0';
```

Comment:
Default module version should remain '1.0.0'

Reply:
Change module version to '1.0.0'